monitoring/datadog_downtime.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Stefan Weller <stefan.weller@asideas.de>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.    If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: datadog_downtime
+short_description: Set/update/cancel active or future downtimes (with optional recurrence) of DataDog monitors by a scope.
+description:
+- "Allows to set/update/cancel active or future downtimes (with optional recurrence) of DataDog monitors (www.datadoghq.com) by providing a scope."
+- "Uses U(http://docs.datadoghq.com/api/#monitors) API."
+- "Uses datadogpy U(https://github.com/DataDog/datadogpy) library -> pip install datadog."
+- "Currently this module has no check mode."
+version_added: "2.2.0"
+author: 'Stefan Weller <stefan.weller@asideas.de>, Github: stefanweller'
+requirements: [datadog]
+options:
+    I(api_key):
+        description: ["DataDog API key."]
+        required: true
+        default: null
+    I(app_key):
+        description: ["DataDog application key."]
+        required: true
+        default: null
+    I(state):
+        description: ["Designated state of the downtime. Currently 'present', 'updated' and 'absent' are supported."]
+        required: true
+        default: null
```

Comment:
Please add 
`choices:` and list present', 'updated' and 'absent', 

again, see other modules for examples